packages/client/components/promptResponse/PromptResponseEditor.tsx

```diff
@@ -56,7 +57,11 @@ const PromptResponseEditor = (props: Props) => {
 
   const onSubmit = ({editor: newEditorState}: EditorEvents['blur']) => {
     setEditing(false)
-    handleSubmit && handleSubmit(newEditorState)
+    const newContent = newEditorState.getJSON()
+
+    if (areEqual(content, newContent)) return
```

Comment:
-1 I think you need to check here for empty and undefined/null to avoid creating an empty post on first blur.

Reply:
done! good point, and we can use `newEditorState.isEmpty` to check the empty input content